src/test/java/thrift/testutil/TagSetBuilder.java

```diff
@@ -0,0 +1,75 @@
+package thrift.testutil;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import thrift.model.tag.Tag;
+import thrift.model.transaction.Transaction;
+
+/**
+ * A utility class to help with building Tag sets.
+ * Example usage: <br>
+ *     {@code Set<Tag> tagSet = new TagSetBuilder("Food").build();}
+ */
+public class TagSetBuilder {
+
+    private final Set<Tag> tagSet;
+
+    /**
+     * Creates a blank TagSetBuilder with no tags
+     */
+    public TagSetBuilder() {
+        this.tagSet = new HashSet<Tag>();
+    }
+
+    /**
+     * Creates a TagSetBuilder with tags made from {@param tagNames}
+     * @param tagNames whose tags are used as reference
+     */
+    public TagSetBuilder(String... tagNames) {
+        this.tagSet = (Stream.of(tagNames).map(Tag::new).collect(Collectors.toSet()));
+    }
+
+    /**
+     * Creates a TagSetBuilder with tags from {@param tagSet}
+     * @param tagSet whose tags are used as reference
```

Comment:
Do leave an empty line between description and parameter section.

Reply:
Added the empty line.